### ISSUE TITLE
Undefined variable k_value in rnaspades.py 3.13

### ISSUE THREAD

**Issue by klamkiew** [ablab/spades#216]
Hello everyone.

When running rnaspades.py with these parameters I get the following NameError:

`rnaspades.py --s1 $R1 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/rnaspades/`

```
  File "/mnt/prostlocal/programs/SPAdes/3.13.0/bin/rnaspades.py", line 1093, in <module>
    main(sys.argv)
  File "/mnt/prostlocal/programs/SPAdes/3.13.0/bin/rnaspades.py", line 725, in main
    k_values = rna_k_values(support, options_storage, dataset_data, log)
  File "/mnt/prostlocal/programs/SPAdes/3.13.0/bin/rnaspades.py", line 575, in rna_k_values
    support.warning("\n" + 'Auto K value (' + str(k_value) + ') is too small, recommended to be at least %d.\n' % (options_storage.RNA_MIN_K))
NameError: global name 'k_value' is not defined
```

Apparently `k_value` does only occur once in the rnaspades.py script, namely in the line given at the error message. I was assuming that this is just a typo and you want to have `upper_k` there. When changing this manually, rnaSPAdes gives me this warning (which looks good to me) and runs without any further errors.

```
== Warning ==  
Auto K value (23) is too small, recommended to be at least 29.



Upper K value is set to 29.

K values to be used: [29]

```

If I am correct, please change the corresponding line, if not, I'd be happy to hear from you what I did wrong :)

Cheers,
Kevin

**Comment by andrewprzh** [comment]
Dear Kevin

Indeed, it's a typo and should be upper_k instead -- thanks a lot for noticing! Will fix it.

Yes, the warning is not not fatal at, just states the the k value will be changed to a higher value. By default, K value is set to about half of the read length, but small k-mers may cause misassemblies, thus we set the minimal possible value to 29. More details can be found here https://www.biorxiv.org/content/early/2018/09/18/420208

Best regards
Andrey

**Comment by asl** [comment]
Fixed in 3.13.1
